flytekit/core/launch_plan.py

```diff
@@ -143,6 +146,9 @@ def get_or_create(
         :param annotations: Optional annotations to attach to executions created by this launch plan.
         :param raw_output_data_config: Optional location of offloaded data for things like S3, etc.
         :param auth_role: Add an auth role if necessary.
+        :param max_parallelism: Controls the maximum number of tasknodes that can be run in parallel for the entire
+            workflow. This is useful to achieve fairness. Note: MapTasks are regarded as one unit, and
+            parallelism/concurrency of MapTasks is independent from this.
```

Comment:
What about nodes in dynamic workflows?

Reply:
this will apply to dynamic workflows too

Reply:
good q! i can update propeller handling to ensure the dynamic workflow inherits the parent's max concurrency

do you think it'd be valuable to expose it as a configurable param separate from the parent workflow?